content/developer/accessibility/index.md

```diff
@@ -0,0 +1,254 @@
+
+
+| HTML5 Element | Equivalent Landmark Role |
+|---|---|
+| aside | complementary |
+| header | banner (if and only if in body tag) |
+| main | main |
+| nav | navigation |
+| section | region (but it also needs either a title, an `aria-labelledby` or `aria-label` attribute) |
+| form | form (but it also needs either a title, an `aria-labelledby` or `aria-label` attribute) |
+| footer | contentinfo (if and only if in body tag) |
+| *no default* | search |
+
+Note that currently `<article/>` and `<section/>` are not used by screen readers. It's still good practice to use them where appropriate. 
+
+## Practical Guide
+
+It's not always easy to find good examples online of accessible widgets. Here's a non exhaustive list of features we need to make widgets accessible.
+
+### HTML
+
+Good, semantic HTML is good for accessibility. The new tags that were added in HTML5 convey meaning to screen readers. Having a good hierarchy of headings will also help the user find the information quickly. 
+
+#### Button or link?
+
+If clicking on this element does something on the page, it's a button. If it goes somewhere, it's a link. 
+
+
+#### Lang
+
+Besides using `lang="en"` on your `<html/>` tag, you should also use it on links to other languages, or any other foreign language elements. Example, if you've got a multilingual site: add it to the links that switch languages.
+
+### Error messages
+
+- `aria-errormessage` contains ID of element with Error message
+- If the message is added after the user has clicked submit or on blur, use `aria-live="assertive"`
+
+Don't overuse `aria-live="assertive"` as it will stop reading anything being read and start reading the new message immediately, but in the case of an important error, it's appropriate.
+
+### Hiding things from the screen reader user
+
+If any information is really irrelevant to a non sighted user, such as an icon that is next to a label that clearly defines that section, you may want to hide it. You can do so by using `aira-hidden="true"`.
+
+### Hiding things from a sighted user
+
+If you want to add additional details to a screen reader user and it's not possible to do so with `aria-label` or `aria-labelledby`, there's a hack that's become a convention that you can use. The traditional name for this class is `sr-only` (screen reader only). 
+
+Here's the class:
+
+```css
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0,0,0,0);
+  border: 0;
+}
+```
+
+Modern screen readers are good at emulating what CSS does. In other words, if you `display: none;` something, it will also be hidden to the screen reader. The above CSS make the element invisible, but it does so by cropping it in a 1px x 1px container. This is a hack, but it's become a convention. Maybe one day we'll have a proper CSS property for this behavior, but we currently don't.
+
```

Comment:
A good example of a use for this "sr-only" is a label(<span> with some text) for a "read more" link